packages/operf-micro/operf-micro.0.2/opam

```diff
@@ -0,0 +1,12 @@
+opam-version: "1.2"
+maintainer: "Pierre Chambart <pierre.chambart@ocamlpro.com>"
+authors: "Pierre Chambart <pierre.chambart@ocamlpro.com>"
+homepage: "http://www.typerex.org/operf-micro.html"
+bug-reports: "http://github.com/OCamlPro/operf-micro/issues"
+license: "GPLv2"
```

Comment:
The OPAM file says the license is GPLv2, but the repository [says](https://github.com/OCamlPro/operf-micro/blob/master/LICENSE) it's MIT.